### ISSUE TITLE
Pressing `Spacebar` when editing metadata text plays and pauses music

### ISSUE THREAD

**Issue by jukibom** [CircuitLord/NotReaper#69]

**Comment by Wolferacing** [comment]
I was going to look into fixing this but I can't replicate the bug.

**Comment by CircuitLord** [comment]
I don't think this happens on the metadata menu. It's on the menu when you make a new Audica, whenever you press space in the input box to type a space it'll play and pause.

**Comment by aggrogahu** [comment]
Yeah, happens with genning a new Audica. Other shortcuts will trigger as well, such as the bookmark shortcuts.
